ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -208,5 +208,21 @@ class BadgeLabelButton: UIButton {
 
     @objc private func badgeValueDidChange() {
         updateBadgeLabel()
+        updateAccessibilityLabel()
+    }
+
+    private func updateAccessibilityLabel() {
+        guard let item = item, let itemAccessibilityLabel = item.accessibilityLabel else {
```

Comment:
is guard let itemaccessibilitylabel correct? sometimes do you want to set it nil.

Reply:
I am sorry but I didn't get your comment. I did this so that we can return early whenever there is no item or if the item's accessibility label is not present.

Reply:
isn't this a change in behavior. before accessibilityLabel = item.accessibilityLabel. it set it to nil if it was nil

Reply:
Done!!